python/lsst/ts/gis/commander.py

```diff
@@ -69,8 +68,8 @@ async def connect(self):
             if not self.simulation_mode:
                 self.tunnel = sshtunnel.open_tunnel(
                     (self.bastion_host, self.bastion_port),
-                    ssh_username="saluser",
-                    password=os.environ["GIS_PASSWORD"],
+                    ssh_username="gis-bastion",
+                    pkey="~/.ssh/id_gis_bastion",
```

Comment:
I think the `ssh_username` and `pkey` values should be part of the configuration and not hard coded.